content/_includes/page/list.njk

```diff
@@ -17,6 +17,8 @@
 {% elif bird and (bird != 'oddbird') %}
   {% set list = collections._post | byBird(bird) %}
   {{ post.by_year(list, collections) }}
+  {% set oss = collections["Open Source"] | byBird(bird) %}
+  {{ post.by_year(oss, collections, 'Open Source', Infinity) }}
```

Comment:
@mirisuzanne So. I removed OSS and Client Work (without a Case Study) from the `_post` collection. I _think_ that removes those from the Resources list, and also from the RSS feed.

But... from what I can tell, there were two other places we were using the `_post` collection:

1. Deciding who from the birds-emeritus has a page to link to. I decided contributing to OSS doesn't really need to require a standalone page. Practically, this makes no difference at the moment.
2. Listing Resources on an individual Bird page. I made some changes so that it list "Open Source" below "Resources"... but I'm not tied to it.

Reply:
I think Sondra has a different plan for OSS list layout, but this is a good placeholder. I moved the `title` arg to the `list()` macro, so we can save `by_year` to do the yearly grouping logic. No need to call it if we're using `Infinity` to avoid it's core feature.